### PR TITLE
enhancement/allegations_acadia_so

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: data/.wrglsync.yaml [hunk 1]
```diff
@@ -576,6 +584,10 @@ repos:
     fileName: fuse/event_terrebonne_so.csv
     primaryKey:
       - event_uid
+  event-acadia-so:
+    fileName: fuse/event_acadia_so.csv
+    primaryKey:
+      - event_uid
```
Comment (pckhoi) [inline data/.wrglsync.yaml:590]: Please merge the `main` branch into this branch and modify the `.wrgl/config.yaml` file instead.

Comment (ayyubibrahimi) [reply data/.wrglsync.yaml:590]: Got it. Thanks.

**Comment by ayyubibrahimi** [comment]
@pckhoi - This is ready for review whenever you have the availability. Thanks!

File: clean/acadia_so_cprr.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+
+def clean():
+    df = pd.read_csv(data_file_path('raw/acadia_so/cprr_acadia_so_2018_2021.csv'))\
+        .pipe(clean_column_names)\
+        .rename(columns={
+            'date': 'receive_date'
+        })\
+        .pipe(clean_dates, ['receive_date'])\
+        .pipe(clean_names)\
```
Comment (pckhoi) [inline clean/acadia_so_cprr.py:39]: Are the names in lower case and with punctuations removed? Even if they are I think it's still good practice to always use the `lib.clean.clean_names` function on the name columns no matter what. That way the names during matching are always uniform.

File: clean/acadia_so_cprr.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        .pipe(clean_allegation)\
+        .pipe(clean_action)\
+        .pipe(set_values, {
+            'agency': 'acadia parish so'
```
Comment (pckhoi) [inline clean/acadia_so_cprr.py:43]: Shouldn't the agency name be in title case? So maybe `Acadia Parish SO` instead?

File: clean/acadia_so_cprr.py (hunk 2 above)
Comment (ayyubibrahimi) [reply clean/acadia_so_cprr.py:39]: Good call. Thank you.

File: clean/acadia_so_cprr.py (hunk 3 above)
Comment (ayyubibrahimi) [reply clean/acadia_so_cprr.py:43]: Yes. I should have held off on branch this until we've standardized agency names re: the POST branch. Thanks.

**Comment by ayyubibrahimi** [comment]
@pckhoi - Please review when you have the time. Thanks.

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**